lib/types.ts

```diff
@@ -78,8 +79,18 @@ export type EventBase = {
    * Source user, group, or room object with information about the source of the event.
    */
   source: EventSource;
+  /**
+   * Webhook event ID, an ID that uniquely identifies a Webhook event, in the form of a ULID string.
+   */
+  webhookEventId: string;
+  /**
+   * Whether the webhook event is a resubmitted one or not
+   */
+  deliveryContext: DeliveryContext;
```

Comment:
It is added in common-properties.
https://developers.line.biz/en/reference/messaging-api/#common-properties